chain/vm/invoker.go

```diff
@@ -47,6 +47,7 @@ func NewInvoker() *Invoker {
 	}
 
 	// add builtInCode using: register(cid, singleton)
+	// NETUPGRADE: register code IDs for v2, etc.
```

Comment:
My plan here is to register code IDs for _both_ v1 and v2 actors, at the same time. As far as I know, this should "just work".

Reply:
Correct, though ideally we'd at least specify minimum / maximum epoch at which a given code CID can be used